api/types/src/ledger_info.rs

```diff
@@ -11,6 +11,7 @@ use serde::{Deserialize, Serialize};
 pub struct LedgerInfo {
     pub chain_id: u8,
     pub epoch: u64,
+    pub round: u64,
```

Comment:
Is this useful to someone who is writing applications around a node?  Is there a set number of rounds in an epoch, or is this to show how much progress has gone through an epoch?

Reply:
This is as close to representation of number of blocks in an epoch as there easily is. (i.e. rounds = blocks + failed_rounds)

But thinking about it more, I am not sure if this it the right place to surface it. I am using it in tests at the moment, so I'll just do something else. If I end up needing it in new validator-health related CLI commands, I might revive this PR. What do you think?

Reply:
Yeah, I'm not sure it needs to be exposed on every API call right now

Reply:
Ahh, Igor makes an interesting point. I might argue we could actually do better here:

Make a status API that returns:
* Current round
* Current version
* Current block number / height
* Failed rounds
* Current block timestamp
* Oldest ledger version
* ChainId

Adding more to each API request... maybe not? I would argue we probably already have too much being returned because we lacked this API. Then we could have the default return -- maybe just the current epoch, version, and timestamp?

Reply:
Oh,  actually that's what I had in mind.  But I cannot reuse much from this pr, so will create a separate one :)